examples/infrastructure/ecs-cluster.yaml

```diff
@@ -335,6 +335,7 @@ Resources:
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/CloudWatchFullAccess
         - arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess
+        - arn:aws:iam::aws:policy/AWSAppMeshEnvoyAccess
```

Comment:
thank you. Did you check other examples as well to make sure they have these permissions?

Reply:
thanks. Added in places where it is missing.